projects/commudle-admin/src/app/feature-modules/labs/components/labs/labs.component.html

```diff
@@ -19,10 +19,27 @@
 
     <section class="body">
       <!-- display list of popular tags and horizontal rule if the search bar is empty -->
-      <div class="popular tags"></div>
+      <div class="popular-tags">
+        <!-- popular tags here -->
+        <div *ngFor="let popTag of popularTags; let i = index">
```

Comment:
Is this index i used somewhere, if not is it required here?

Reply:
just keeping Tag instead of popTag would help